### PR TITLE
[MI-2449]:Fixed bugs found by QA

### PR DESCRIPTION
**Pull request by Kshitij-Katiyar**

### CODE REVIEW

File: server/command.go [hunk 1]
```diff
@@ -444,6 +444,11 @@ func deleteConfig(p *Plugin, context *model.CommandArgs, args ...string) *model.
 		return &model.CommandResponse{}
 	}
 
+	if len(args) == 0 {
+		response := executeConfluenceDefault(p, context, args...)
+		return response
```
Comment (ayusht2810) [inline server/command.go]: can't we directly return this, instead of creating a new variable?

File: server/serializer/channel_subscription.go [hunk 2]
```diff
@@ -21,9 +21,9 @@ const (
 	SubscriptionTypeSpace = "space_subscription"
 	SubscriptionTypePage  = "page_subscription"
 
-	aliasAlreadyExist       = "a subscription with the same name already exists in this channel"
-	urlSpaceKeyAlreadyExist = "a subscription with the same url and space key already exists in this channel"
-	urlPageIDAlreadyExist   = "a subscription with the same url and page id already exists in this channel"
+	aliasAlreadyExist       = "subscription with the same name already exists in this channel"
+	urlSpaceKeyAlreadyExist = "subscription with the same url and space key already exists in this channel"
+	urlPageIDAlreadyExist   = "subscription with the same url and page id already exists in this channel"
```
Comment (ayusht2810) [inline server/serializer/channel_subscription.go]:
```suggestion
	urlSpaceKeyAlreadyExist = "subscription with the same URL and space key already exists in this channel"
	urlPageIDAlreadyExist   = "subscription with the same URL and page id already exists in this channel"
```

**Review by ayusht2810 — APPROVED** [review]

File: server/service/save_subscription.go [hunk 3]
```diff
@@ -9,12 +9,7 @@ import (
 	"github.com/mattermost/mattermost-plugin-confluence/server/store"
 )
 
-const (
-	generalSaveError        = "an error occurred attempting to save a subscription"
-	aliasAlreadyExist       = "a subscription with the same name already exists in this channel"
-	urlSpaceKeyAlreadyExist = "a subscription with the same url and space key already exists in this channel"
-	urlPageIDAlreadyExist   = "a subscription with the same url and page id already exists in this channel"
-)
+const generalSaveError = "an error occurred attempting to save a subscription"
```
Comment (avas27JTG) [inline server/service/save_subscription.go]:
```suggestion
const generalSaveError = "an error occurred while attempting to save a subscription"
```

File: server/command.go [hunk 4]
```diff
@@ -203,12 +187,25 @@ func getAutoCompleteData(showMigrateCommands bool) *model.AutocompleteData {
 
 	confluence.AddCommand(config)
 
+	connect := model.NewAutocompleteData("connect", "", "Connect your Mattermost account to your Confluence account")
+	confluence.AddCommand(connect)
+
+	disconnect := model.NewAutocompleteData("disconnect", "", "Disconnect your Mattermost account to your Confluence account")
```
Comment (avas27JTG) [inline server/command.go]:
```suggestion
	disconnect := model.NewAutocompleteData("disconnect", "", "Disconnect your Mattermost account from your Confluence account")
```

**Review by avas27JTG — APPROVED** [review]

**Pull request merged**